terraso_backend/apps/audit_logs/models.py

```diff
@@ -26,7 +27,7 @@ class Log(models.Model):
     user = models.ForeignKey(
         settings.AUTH_USER_MODEL, on_delete=models.PROTECT, verbose_name="user"
```

Comment:
question: why is `AUTH_USER_MODEL` being used here? (i see `User` being used for `ForeignKey`s elsewhere, e.g. in the `Group` model)

Reply:
I think that working with models is considered a good practice
https://learndjango.com/tutorials/django-best-practices-referencing-user-model

Reply:
i see, that makes sense, thank you. we should file a backlog issue to migrate the other models if so since they should be consistent, but that shouldn't be part of this PR. @josebui do you have any input on using `settings.AUTH_USER_MODEL` for the other models?